config.go

```diff
@@ -54,13 +56,28 @@ func FromEnv() (*Config, error) {
 		printFailedOnly = true
 	}
 
+	enableRetries := true
+	if getEnv("ENABLE_RETRIES", "true") == "false" {
+		enableRetries = false
+	}
+
+	retryCount, err := strconv.Atoi(getEnv("DEFAULT_RETRY_COUNT", "3"))
```

Comment:
Or should this be 2?

Reply:
good catch! Let's go with 2 as the default